### ISSUE TITLE
Feature request: provide a gradio gui

### ISSUE THREAD

**Issue by ljleb** [s1dlx/sd-webui-bayesian-merger#9]
I think it would make this extension more accessible to provide a gradio gui that calls the cli script.

**Comment by s1dlx** [comment]
I agree that a GUI would help more people using the script. At the moment this is low priority before I sort out other aspects of the extension. 

However, PRs are welcome on this subject
